util/fipstools/acvp/modulewrapper/modulewrapper.cc

```diff
@@ -2206,18 +2268,26 @@ static struct {
   bool (*handler)(const Span<const uint8_t> args[], ReplyCallback write_reply);
 } kFunctions[] = {
     {"getConfig", 0, GetConfig},
-    {"SHA-1", 1, Hash<SHA1, SHA_DIGEST_LENGTH>},
-    {"SHA2-224", 1, Hash<SHA224, SHA224_DIGEST_LENGTH>},
-    {"SHA2-256", 1, Hash<SHA256, SHA256_DIGEST_LENGTH>},
-    {"SHA2-384", 1, Hash<SHA384, SHA384_DIGEST_LENGTH>},
-    {"SHA2-512", 1, Hash<SHA512, SHA512_DIGEST_LENGTH>},
-    {"SHA2-512/256", 1, Hash<SHA512_256, SHA512_256_DIGEST_LENGTH>},
+    {"SHA-1", 1, Hash<SHA_DIGEST_LENGTH, EVP_sha1>},
+    {"SHA2-224", 1, Hash<SHA224_DIGEST_LENGTH, EVP_sha224>},
+    {"SHA2-256", 1, Hash<SHA256_DIGEST_LENGTH, EVP_sha256>},
+    {"SHA2-384", 1, Hash<SHA384_DIGEST_LENGTH, EVP_sha384>},
+    {"SHA2-512", 1, Hash<SHA512_DIGEST_LENGTH, EVP_sha512>},
+    {"SHA2-512/256", 1, Hash<SHA512_256_DIGEST_LENGTH, EVP_sha512_256>},
+    {"SHA3-224", 1, Hash<SHA224_DIGEST_LENGTH, EVP_sha3_224>},
+    {"SHA3-256", 1, Hash<SHA256_DIGEST_LENGTH, EVP_sha3_256>},
+    {"SHA3-384", 1, Hash<SHA384_DIGEST_LENGTH, EVP_sha3_384>},
+    {"SHA3-512", 1, Hash<SHA512_DIGEST_LENGTH, EVP_sha3_512>},
     {"SHA-1/MCT", 1, HashMCT<SHA1, SHA_DIGEST_LENGTH>},
     {"SHA2-224/MCT", 1, HashMCT<SHA224, SHA224_DIGEST_LENGTH>},
     {"SHA2-256/MCT", 1, HashMCT<SHA256, SHA256_DIGEST_LENGTH>},
     {"SHA2-384/MCT", 1, HashMCT<SHA384, SHA384_DIGEST_LENGTH>},
     {"SHA2-512/MCT", 1, HashMCT<SHA512, SHA512_DIGEST_LENGTH>},
     {"SHA2-512/256/MCT", 1, HashMCT<SHA512_256, SHA512_256_DIGEST_LENGTH>},
+    {"SHA3-224/MCT", 1, HashMCTSha3<SHA224_DIGEST_LENGTH, EVP_sha3_224>},
```

Comment:
Minor styling nit, but can we swap `SHA224_DIGEST_LENGTH` and `EVP_sha3_224` to align with the ones in middle?